### ISSUE TITLE
Translate emails

### ISSUE THREAD

**Issue by TCOTC** [thunderbird/thunderbird-android#7482]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same feature request.
- [X] I will describe the problem with as much detail as possible.
- [X] This issue only contains a request for one single feature, **not** multiple (related) features.

### App version

6.714

### Problem you are trying to solve

有时候我需要将整个邮件内容翻译为我的母语，而我目前没有找到方便的方法，我只能通过两种麻烦的方式解决这个问题：

1. 将邮件内容复制到其他翻译软件，然后翻译。
2. 使用 Chrome 打开邮件网站（例如 mail.google.com ），然后使用 Google Translate 翻译页面。

---

Sometimes I need to translate the entire email content into my native language, and I haven't found a convenient way at the moment. I can only solve this problem in two troublesome ways:

1. Copy the email content to other translation software and then translate it.
2. Use Chrome to open the mail website (such as mail.google.com), and then use Google Translate to translate the page.

### Suggested solution

点击一个按钮，然后在阅读邮件时直接显示翻译后的内容，就像在 Chrome 上运行的 Google Translate ，不需要借助其他软件。

---

Click a button and then display the translated content directly when reading an email, just like Google Translate running on Chrome, without the need for additional software.

### Screenshots / Drawings / Technical details

就像这个：

---

Like this:

![image](https://github.com/thunderbird/thunderbird-android/assets/78434827/c1766188-5c32-4c8e-96c5-81c3fc8f488e)

**Comment by kewisch** [comment]
This sounds like a great idea to post to https://connect.mozilla.org/t5/ideas/idb-p/ideas/label-name/thunderbird%20android, would you mind repeating it there? Using the local Firefox translation SLM would be pretty cool.

**Comment by kewisch** [comment]
Yes, small language model. When you've posted the idea there I'd appreciate if you could link it here and close the issue, this way we can centralize the idea in one place.

**Comment by kewisch** [comment]
Yes, we've already changed the file an issue process to bug reports only. There are still a bunch of enhancements on the bug tracker for legacy reasons though :) Closing this issue now that we have one one connect, thank you!

**Comment by kolAflash** [comment]
FTR:  
_Support built-in Firefox Translations in Thunderbird_  
https://bugzilla.mozilla.org/show_bug.cgi?id=1823007
